### PR TITLE
Restaurant Review Screen

### PR DESCRIPTION
**Pull request by Nap1400**
image

### CODE REVIEW

**Review by carizzuti — APPROVED** [review]
Screen looks good, next we should work on configuring axios for this screen. Will be a post request to the review endpoint.

**Comment by Dulaya** [comment]
- In RestaurantReview.view.tsx:
   - You have a lot of unused imports. Delete them if you're not using the imports.
   - SafeAreaView should be import from 'react-native'
   - Don't use inline styling.
   - You should get your image from your end point. Also pizza_hut_logo.png doesn't exist. 
   - The button needs rounder borderRadius.
   - You need to use Header inside the Container. Make sure the back button navigate to previous screen. 

- In RestaurantReview.type.ts:
   - You also have a lot of unsed imports. Delete them if they're not being used.

- In RestaurantReview.tsx:
   - You need to pass your data from here to the RestaurantReview.view.tsx as props instead of hard-coding them there.

**Review by Dulaya — CHANGES_REQUESTED** [review]
- In RestaurantReview.view.tsx:
   - You have a lot of unused imports. Delete them if you're not using the imports.
   - SafeAreaView should be import from 'react-native'
   - Don't use inline styling.
   - You should get your image from your end point. Also pizza_hut_logo.png doesn't exist. 
   - The button needs rounder borderRadius.
   - You need to use Header inside the Container. Make sure the back button navigate to previous screen. 

- In RestaurantReview.type.ts:
   - You also have a lot of unsed imports. Delete them if they're not being used.

- In RestaurantReview.tsx:
   - You need to pass your data from here to the RestaurantReview.view.tsx as props instead of hard-coding them there.

File: mobile/src/screens/restaurantReview/RestaurantReview.style.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import { StyleSheet } from "react-native";
+import { color, font, size } from "../../theme";
+
+export const styles = StyleSheet.create({
+    buttonContainer: {
```
Comment (lasharela) [inline mobile/src/screens/restaurantReview/RestaurantReview.style.ts:5]: review this

File: mobile/src/screens/restaurantReview/RestaurantReview.view.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    <Container>
+      <View
+        style={{
+          alignItems: 'center',
+          justifyContent: 'center',
+          paddingTop: size.xxl,
+        }}>
+        <Image 
+            source={require('../../assets/images/pizza_hut_logo.png')}
+            style={{
```
Comment (lasharela) [inline mobile/src/screens/restaurantReview/RestaurantReview.view.tsx:37]: no inline styles

File: mobile/src/screens/restaurantReview/RestaurantReview.view.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  return (
+    <Container>
+      <View
+        style={{
+          alignItems: 'center',
+          justifyContent: 'center',
+          paddingTop: size.xxl,
+        }}>
+        <Image 
+            source={require('../../assets/images/pizza_hut_logo.png')}
```
Comment (lasharela) [inline mobile/src/screens/restaurantReview/RestaurantReview.view.tsx:36]: create index.ts inside of assets

File: mobile/src/screens/restaurantReview/RestaurantReview.view.tsx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+                borderRadius: 10
+            }}
+        />
+      </View>
+      <Text style={styles.title}>
+        How was your last order from {restaurant}?
+      </Text>
+      <RatingStars />
+      <TextInput
+        type={'regular'}
```
Comment (lasharela) [inline mobile/src/screens/restaurantReview/RestaurantReview.view.tsx:49]: remove this

**Review by lasharela — APPROVED** [review]

**Review by raynesmike — APPROVED** [review]

File: mobile/src/screens/restaurantReview/RestaurantReview.view.tsx [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+import {styles} from './RestaurantReview.style';
+import {TRestaurantReviewView} from './RestaurantReview.type';
+
+export const RestaurantReviewScreenView: FC<TRestaurantReviewView> = ({
+  restaurant = 'Pizza Hut',
+}) => {
+  const [review, setReview] = useState();
+  return (
+    <Container>
+      <SafeAreaView>
```
Comment (raynesmike) [inline mobile/src/screens/restaurantReview/RestaurantReview.view.tsx:19]: c

File: mobile/src/screens/restaurantReview/RestaurantReview.view.tsx [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+        <View
+          style={{
```
Comment (lasharela) [inline mobile/src/screens/restaurantReview/RestaurantReview.view.tsx:21]: no inline styles

File: mobile/src/screens/restaurantReview/RestaurantReview.view.tsx [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+            alignItems: 'center',
+            justifyContent: 'center',
+            paddingTop: size.xxl,
+          }}>
+          <Image 
+              source={require('../../assets/images/pizza_hut_logo.png')}
+              style={{
+                  height: 100,
```
Comment (lasharela) [inline mobile/src/screens/restaurantReview/RestaurantReview.view.tsx:29]: same

**Review by lasharela — CHANGES_REQUESTED** [review]

**Review by Josetf2396 — APPROVED** [review]

**Review by Dulaya — APPROVED** [review]

**Pull request merged**